x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -215,19 +215,20 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
     setEventsLoading,
   ]);
 
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const openAlertActionComponent = (
-    <EuiContextMenuItem
-      key="open-alert"
-      aria-label="Open alert"
-      data-test-subj="open-alert-status"
-      id={FILTER_OPEN}
-      onClick={openAlertActionOnClick}
-      disabled={!hasIndexUpdateDelete && !hasIndexMaintenance}
-    >
-      <EuiText size="m">{i18n.ACTION_OPEN_ALERT}</EuiText>
-    </EuiContextMenuItem>
-  );
+  const openAlertActionComponent = useMemo(() => {
+    return (
+      <EuiContextMenuItem
+        key="open-alert"
+        aria-label="Open alert"
+        data-test-subj="open-alert-status"
+        id={FILTER_OPEN}
+        onClick={openAlertActionOnClick}
+        disabled={!hasIndexUpdateDelete && !hasIndexMaintenance}
```

Comment:
is it still disabled or isDisabled?

Reply:
For ```EuiContextMenu``` and most other components, both disabled and isDisabled will work. However for EuiButtonIcon ONLY isDisabled will work, which is the cause of the bug

Reply:
I asked it because if EUI deprecated `disabled` for `EuiButtonIcon` might be a good idea to do the same every where. I will talk to the EUI folks before we do anything. Thank you to let me know.